content/en/docs/distributions/charmed/_index.md

```diff
@@ -1,14 +1,30 @@
+[Charmed Kubeflow](https://charmed-kubeflow.io/) from [Canonical](https://www.canonical.com/) delivers a powerful, sophisticated end-to-end MLOps platform which you can deploy using any conformant Kubernetes distribution. The solution enables optimised AI training and modelling, in a robust and automated way, allowing data scientists to focus on AI/ML projects, not tools. The enterprise-ready platform is available backed with 24/7 support, expert set-up services, and managed services with a service level agreement (SLA). With a growing ecosystem of powerful extensions and integrations including MLFlow or Seldon, the Charmed Kubeflow solution amplifies productivity levels for data scientists and machine learning engineers working with advanced analytics and AI.
 
-Features include:
-* _Ecosystem:_ A growing ecosystem of extensions and integrations, such as [MLFlow](https://mlflow.org) to help scale up your data science initiatives.
-* _Any cloud_: Charmed Kubeflow is a fully portable solution for any cloud, including on-premise Kubernetes. Train your teams once to work anywhere.
+Charmed Kubeflow is Canonical’s version of the upstream Kubeflow project. It uses charms, allowing components to easily connect and work effectively for different use cases. Main features include:
+
+* _Free to use_: Charmed Kubeflow is offered as free, open-source software. You don’t need a support agreement or license to deploy Charmed Kubeflow.
+* _Integrated_: A growing ecosystem of extensions and integrations, such as [MLFlow](https://mlflow.org/) to help scale up your data science initiatives.
+* _Available_: Charmed Kubeflow is a fully portable solution for any cloud, including on-premise Kubernetes. Train your teams once to work anywhere.
 * _Supported_: [24/7 support](https://ubuntu.com/support), [professional services](https://ubuntu.com/ai/services) and [managed services](https://ubuntu.com/managed/apps) with guaranteed SLA are available.
-* _Free to use_: Charmed Kubeflow is offered as free, open source software. You don't need a support agreement or license to deploy Charmed Kubeflow.
+* _Community-driven:_ Charmed Kubeflow is an [open-source product d](https://github.com/canonical/bundle-kubeflow)riven by the community’s [feedback](https://discourse.charmhub.io/tag/kubeflow). You can always contribute or get our team’s support.
+
+---
+
+[Charmed Kubeflow deployment guide](https://charmed-kubeflow.io/docs/quickstart)
+Instructions for Kubeflow deployment with Kubeflow Charmed Operators
+
+[Charmed Kubeflow upgrade guide](https://charmed-kubeflow.io/docs/upgrade)
+Instructions to update to the latest version of Charmed Kubeflow.
+
+[Charmed Kubeflow tutorials](https://charmed-kubeflow.io/tutorials)
+A library of fun tutorials once you get started with Kubeflow
+
+[Charmed Kubeflow documentation](https://charmed-kubeflow.io/docs)
+Instructions to use different components of Charmed Kubeflow
```

Comment:
"Official documentation" seems more appropriate as the link doesn't really point to instructions.